phpunit/functional/AuthTest.php

```diff
@@ -215,4 +215,23 @@ public function testPasswordCostUpdate(): void
         preg_match('/\$2y\$(\d+)\$.+/', $user->fields['password'], $new_cost);
         $this->assertGreaterThan(5, (int) $new_cost[1]);
     }
+
+    public function testRememberMeLastLogin(): void
+    {
+        $user = getItemByTypeName(User::class, 'post-only');
+        $token = $user->getAuthToken('cookie_token', true);
+        $this->assertnull($user->fields['last_login']);
+
+        $_COOKIE[session_name() . '_rememberme'] = json_encode([
+            $user->getID(),
+            $token,
+        ]);
+
+        //login using remember_me cookie just set
+        $this->assertTrue((new \Auth())->login('', ''));
+
+        //check if last_login is now set
+        $this->assertTrue($user->getFromDB($user->getID()));
+        $this->assertNotNull($user->fields['last_login']);
```

Comment:
Maybe should we unset the cookie?
```suggestion
        $this->assertNotNull($user->fields['last_login']);
        
        unset($_COOKIE[session_name() . '_rememberme']);
```